tests/ci/cherry_pick.py

```diff
@@ -67,15 +67,17 @@ class ReleaseBranch:
 ### Note
 
 This pull-request will be merged automatically as it reaches the mergeable state, \
-**do not merge it manually**.
+**do not merge it manually**. It's 100% safe, but completely meaningless.
 
 ### If the PR was closed and then reopened
 
-If it stuck, check {pr_url} for `{backport_created_label}` and delete it if \
-necessary. Manually merging will do nothing, since `{backport_created_label}` \
-prevents the original PR {pr_url} from being processed.
+If it stuck (e.g. for a day), check {pr_url} for `{backport_created_label}` *label* and \
+delete it if necessary. Manually merging will do nothing, since \
```

Comment:
Maybe this?

```
This is necessary as label `{backport_created_label}' otherwise prevents the original PR {pr_url} from being processed. Please do merge the PR manually in such a situation as this will screw up the process.
```